### PR TITLE
chore(docs): update prerequisites

### PR DESCRIPTION
**Pull request by niladrih**
Signed-off-by: Niladri Halder <niladri.halder@mayadata.io>

This PR extends the prerequisites to mention raw block devices

### CODE REVIEW

File: docs/quick.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ meets the following prerequisites:
 | Ubuntu/Debian   | open-iscsi            |  <ul><li>sudo apt install open-iscsi -y</li><li>sudo systemctl enable --now iscsid</li></ui>| sudo systemctl status iscsid.service |
 | RancherOS        | open-iscsi            |  <ul><li>sudo ros s enable open-iscsi</li><li>sudo ros s up open-iscsi</li></ui>| ros service list iscsi |
 
-3. You have disks attached to nodes to provision storage.
+3. You have disks attached to nodes to provision storage. The disks MUST not have any filesystem and the disks MUST not be mounted on the Node. cStor requires raw block devices. You can use the `lsblk` command to check if the disks have a filesystem or if the disk is mounted.
```
Comment (mittachaitu) [inline docs/quick.md]: `lsblk -fa` shows the filesystem information

```suggestion
3. You have disks attached to nodes to provision storage. The disks MUST not have any filesystem and the disks MUST not be mounted on the Node. cStor requires raw block devices. You can use the `lsblk -fa` command to check if the disks have a filesystem or if the disk is mounted.
```

**Review by mittachaitu — COMMENTED** [review]
Looks good but provided few comments

File: docs/quick.md (hunk 1 above)
Comment (pawanpraka1) [inline docs/quick.md]: to provision *the* storage.

**Review by mittachaitu — APPROVED** [review]
LGTM

**Review by kmova — APPROVED** [review]

**Pull request merged**